_shared/project/tox.ini

```diff
@@ -135,6 +139,13 @@ commands =
     {% endif %}
 {% else %}
     dev: {posargs:ipython --classic --no-banner --no-confirm-exit}
+    {% if cookiecutter.get("linter") == "ruff" %}
+    format: ruff check --select I --fix src tests bin
+    format: ruff format src tests bin
+    checkformatting: ruff check --select I src tests bin
+    checkformatting: ruff format --check src tests bin
+    lint: ruff check --preview -q src tests bin
+    {% else %}
```

Comment:
The commands are all duplicated twice in the `tox.ini` template: once for apps and once for packages. Here I'm adding Ruff support to the commands section for packages. (This could probably be refactored to remove the duplication.)